pkg/apis/autoscaling/annotation_validation.go

```diff
@@ -37,11 +37,12 @@ func getIntGE0(m map[string]string, k string) (int64, *apis.FieldError) {
 	return i, nil
 }
 
-func ValidateAnnotations(anns map[string]string) *apis.FieldError {
+// ValidateAnnotations verifies the autoscaling annotations.
+func ValidateAnnotations(allowInitScaleZero bool, anns map[string]string) *apis.FieldError {
```

Comment:
Should the context get passed in here? With more switches, we'd otherwise grow the signature.

Reply:
@markusthoemmes there's a cyclic package import problem if we do so because we have to import `knative.dev/serving/pkg/apis/config` in `knative.dev/serving/pkg/apis/autoscaling` to get the configmap.
```
can't load package: import cycle not allowed
package knative.dev/serving/pkg/apis/autoscaling
	imports knative.dev/serving/pkg/apis/config
	imports knative.dev/serving/pkg/autoscaler/config
	imports knative.dev/serving/pkg/apis/autoscaling
```
Is there any workaround for this problem?

Reply:
Hmm, probably not an easy one 🤔. Maybe it's fine for now 🤷‍♂️